man/systemd.network.xml

```diff
@@ -1890,6 +1890,15 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SocketPriority=</varname></term>
+        <listitem>
+          <para>An integer value assigned as the linux internal socket priority of the
+          raw IP socket used to send initial DHCPv4 messages. If not set, the default raw socket priority
+          is unchanged.</para>
```

Comment:
must specify valid range and default

Reply:
should also mention that this exposes SO_PRIORITY plus a man page reference to where that's documented.

Reply:
Fixed in the squashed commit.